runners/wikiConsole/templates/intro.asciidoc

```diff
@@ -0,0 +1,9 @@
+== <%= title; %>
+<% if(subtitle){ %>=== <%= subtitle; %> <% } %>
+<%= description; %>
+
+<% if(name){ %>
+The definition of each step of this tutorial can be found at https://github.com/devonfw-tutorials/tutorials/tree/main/<%= name; %>. <% } %>
+
+Feel free to report any errors to us or fix them yourself. Errors can be reported by creating an issue in the https://github.com/devonfw-tutorials/tutorials[tutorials repository]. To fix the error fork the repository and create a pull request. Errors in the wiki can be reported and fixed in the https://github.com/devonfw-tutorials/tutorial-compiler[Tutorial-compiler repository].
```

Comment:
You can use the direct link to the issues site here
https://github.com/devonfw-tutorials/tutorials/issues
And add this section also to the katacoda intro template